### DISCUSSION TITLE
Create Windows container image already bootstrapped for vcpkg

### DISCUSSION THREAD

**Discussion by cglong** [microsoft/vcpkg#25904] [Ideas]
****

**Is your feature request related to a problem? Please describe.**
The vcpkg bootstrap process of our build pipeline takes a majority of the total time and is also the most likely to fail. We've encountered a blocking issue where PowerShell Core often failed to install, blocking our builds.

**Proposed solution**
We'd like a Windows container with all of vcpkg's dependencies (e.g., PowerShell Core, Cygwin, etc.) already installed.

**Describe alternatives you've considered**
Today, we're explicitly downloading and installing PowerShell Core before bootstrapping vcpkg. We've also considered maintaining our own container with the dependencies preinstalled. Both of these options will require continued maintenance on our part to ensure the dependencies are kept up to date.

**Additional context**
N/A

**Comment by JackBoosY** [comment]
Maybe you need https://github.com/microsoft/vcpkg-tool#vcpkg-ce-configure-environment--artifacts ?
